lake/ztests/query-parse-error.yaml

```diff
@@ -34,34 +34,34 @@ outputs:
   - name: stderr
     data: |
       =1=
-      error parsing SuperPipe at line 1, column 11:
+      error parsing SuperPipe at line 1, column 12:
       from test \ count()
-            === ^ ===
+             === ^ ===
```

Comment:
Pointing at the space after the offending token is confusing and we should fix that.

Reply:
@mccanne and I spoke about this and it's something we can look into later.